src/bvals/comms/boundary_communication.cpp

```diff
@@ -366,18 +366,18 @@ TaskStatus ProlongateBounds(std::shared_ptr<MeshData<Real>> &md) {
 
   auto [rebuild, nbound] = CheckReceiveBufferCacheForRebuild<bound_type, false>(md);
 
-  if (rebuild) {
```

Comment:
Is this stuff no longer necessary?

Reply:
That should be removed from this PR since it isn't relevant to the solver (it got included in the PR by mistake), but I *think* the change may actually be necessary. There was a bug I ran into a few weeks ago that was fixed by this change, but I never was able to fully wrap my head around what was going on.